packager/packager.js

```diff
@@ -34,6 +34,10 @@ var parseCommandLine = require('./parseCommandLine.js');
 var webSocketProxy = require('./webSocketProxy.js');
 
 var options = parseCommandLine([{
+  command: 'origin',
+  default: 'http://localhost',
+  type: 'string',
+}, {
```

Comment:
@ide I wanted to model it after HTML's  `location.origin`. But you are right, `origin` includes the port number. Not sure whether `hostname` is a better name and it doesn't include the `protocol`. Is there a better name than `origin`? I went through the [HTML location properties](http://www.w3schools.com/jsref/obj_location.asp). But nothing seems to fit...